src/map/battle.cpp

```diff
@@ -4613,6 +4608,11 @@ struct Damage battle_attack_sc_bonus(struct Damage wd, struct block_list *src, s
 			RE_ALLATK_ADDRATE(wd, sc->data[SC_GVG_GIANT]->val3);
 		}
 
+		if (sc->data[SC_EXEEDBREAK]) {
+			ATK_ADDRATE(wd.damage, wd.damage2, sc->data[SC_EXEEDBREAK]->val2);
+			RE_ALLATK_ADDRATE(wd, sc->data[SC_EXEEDBREAK]->val2);
+		}
+
```

Comment:
This line amplified all damages including skill damage where it shouldn't be. Exceed Break should only increase normal attack.